ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -163,6 +163,15 @@ Item {
             }
         }
 
+        StatusSettingsLineButton {
+            text: "GIF"
+            isSwitch: true
+            switchChecked: appSettings.isGifEnabled
+            onClicked: {
+                appSettings.isGifEnabled = !appSettings.isGifEnabled
+            }
+        }
```

Comment:
I think this switch will later live in the Privacy section, but you don't need to change it now, since it's not decided yet

Reply:
Maybe let's switch all setting together